src/nxdoc/nuxeo-server/installation/tomcat-server-zip/installing-and-setting-up-related-software.md

```diff
@@ -544,7 +544,7 @@ The Nuxeo Platform modules use external software for some features. They need to
 On a naked default Nuxeo Platform, you need the following software:
 
 - For web preview of office documents: LibreOffice and pdftohtml
-  - LibreOffice (version >= 5): converts office file into PDF
+  - LibreOffice (version >= **6.4.7**): converts office file into PDF
```

Comment:
We're gonna upgrade LibreOffice into our Docker Image to version 7.1.0, we should align the version here.